repo/packages/M/mongodb-admin/0/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "packagingVersion": "3.0",
+  "minDcosReleaseVersion": "1.7",
+  "name": "mongodb-admin",
+  "version": "0.0.20-0.1",
```

Comment:
@pyronicide I have seen some services with version number starting from 0.0.1 and few with 0.1. Do we have a standard around this ?